apps/test/unit/dance/danceReduxTest.ts

```diff
@@ -104,7 +104,7 @@ describe('danceRedux', function () {
       userManifest = 'user-manifest';
       queryParams.returns(userManifest);
 
-      songManifest = {};
+      songManifest = [{id: 'a'}, {id: 'b'}, {id: 'c'}];
```

Comment:
This test seems like it was originally wrong - `songManifest` appears to be an array, and I think we just weren't checking against it.

Anyway, made it an array and populated some data to use in the new test later on.

Reply:
Thanks for fixing this!